DESIGNME.md

```diff
@@ -0,0 +1,119 @@
+### Proto-Readme
+
+I had a long talk with Colin the other night, and agreed to take a few notes to
+start shaping in to a proper `README`. They are here, for your delectation,
+review, and commentary:
+
+#### D3 Over Time
+
+**Step One**
+
+1.  We are reclaiming SVG as markup
+2.  We are making looping over data explicit
+    1. We are doing away with `enter`
+3.  Do away with d3's entire dom model
+    We couldn't do this with jQuery; now we have React.
+
+**Step Two**
+
+1.  Everybody loves to copy paste
+    When you start with d3, you get a long-ass list of scatter plots and you copy
+    paste one and pray it works.
+2.  Functionality is locked up in bl.ocks and gists -- wrong. Gists aren't the
+    right format for code of this kind.
+3.  With d3-as-it-has-been, functionality is not componentized. 
+4.  So we'll make: components as discrete, manageable, configurable pieces
+5.  There are a lot of other charting libraries out there; this one is
+    different, especially if you're doing this from the beginning
+6.  Sensible defaults
+
+
+### Design / Welcome to the Neighborhood Questions
+
+**Preface**: This is not my first rodeo; it's not even really my first
+  JavaScript Rodeo, really. But: I'm a server-dweller. I work a lot in compiled
+  languages, with a heavy dose of Python. Some of these thoughts and questions
+  might be pretty far on the n00b end of the spectrum. Learn me a thing. I can
+  keep up.
+
+
+#### Question One: Composition
+
+How granular do we hope to get with the composed nature of `react`? I've worked
+with `react` some; I use composition-the-programming-technique a freaking
+lot. When somebody says, "let's use `react` to replace `d3`'s hideous DOM
+model", what I think of is a heirarchy a little like:
+
+```
+module: chart
+constituent modules:
+- line
+- grid (made of line again)
+- point (made of a shape and an optional label)
+- legend
+```
+
+But now we're getting in to the business of using `react` to replace `d3`'s
+logic, instead of just it's math. Another, maybe/probably better way to do this
+would look more like going down Michael Bostock's npm profile page and making
+one `victory`  module for every `d3` module. Chart, then, would just be:
+
+```
+module: chart
+```
+
+I'm kinda hoping we're planning on the second one. Y'all?
```

Comment:
I'm kind of hoping for the first one, so we can re-use things like `point` and `legend` in more places than `chart`.  Maybe some number of these things just belong in a repo called `victory-base` or `victory-garden` as you suggested earlier, since probably no one will be wanting to use `point` all on its lonesome.